src/illuminatio/rule.py

```diff
@@ -66,8 +87,11 @@ def from_network_policy(cls, net_pol: k8s.client.V1NetworkPolicy):
         return cls(concerns, allowed)
 
 
-# helper function:
+# helper function
 def build_connections(verb, target, ports):
+    """
+    helper function to build Connection tuples
```

Comment:
Start with a capital letter to match the rest of the docstrings (see: https://www.python.org/dev/peps/pep-0257/)